src/slugline.scss

```diff
@@ -39,3 +31,14 @@ h5,
 h6 {
   text-transform: uppercase;
 }
+
+// Form styling
+input.form-control {
+  border-top: none;
+  border-left: none;
+  border-right: none;
```

Comment:
This breaks input groups:
![image](https://user-images.githubusercontent.com/7224751/81082100-cd73ba00-8ec0-11ea-9abc-884b53db48bd.png)

Reply:
Yeah, not sure how to fix that. Maybe just make it a separate button.

Reply:
I believe the input group is encased in an `.input-group` class or something like that. We could probably just remove all but the bottom border.

Reply:
![image](https://user-images.githubusercontent.com/7670245/81113675-a0d69700-8eee-11ea-9cca-62e09b2961d3.png)

Tried it, and it honestly looks like a floating eye. Think we need some kind of differentiation on the left, but with the left and bottom border it looks weird too.

Reply:
Came up with this -- thoughts?
![image](https://user-images.githubusercontent.com/7224751/81118286-a1732b80-8ef6-11ea-9684-c2e1c92b8d71.png)

Reply:
Looks better, but I still think we need some way to distinguish the eye as a button and not just an icon. Maybe another background color?

I do like the background color on the input though, it matches the "subscribe to delivery" section we have at the back of some issues, which is the only reference we have for form inputs.

Reply:
Yeah, the disabled aspect is not appealing...I'm thinking we can leave it as is right now, honestly I'm fine with the floating eye look --- we can also have the bottom border match colours in CSS later on.

Alternatively, we can keep the default Bootstrap styling.

Reply:
So I just consulted Material UI, (which is what we're trying to copy here anyyway), and...
they just went with the floating eye. Let's go with it.